### ISSUE TITLE
No pytest topology mark in VxLAN-CRM testcase

### ISSUE THREAD

**Issue by echuawu** [sonic-net/sonic-mgmt#8514]
#### Description

There is no pytestmark specified in tests/vxlan/test_vxlan_crm.py, which leads to check failure when running this script.

#### Steps to reproduce the issue:
Run script

#### Describe the results you received:
Script run failure due to no pytest topology mark

#### Describe the results you expected:
The script could be run

**Comment by echuawu** [comment]
Hi @rraghav-cisco , does this script must run at cisco-8000 asic?

**Comment by rraghav-cisco** [comment]
@echuawu : No, there are other platforms that support the Vxlan feature, but I am not sure about their crm numbers.

**Comment by echuawu** [comment]
@rraghav-cisco , I met this failure, it would raise error if its asic_type is not 'cisco-8000'

        asic_type = duthosts[rand_one_dut_hostname].facts["asic_type"]
        if asic_type == "cisco-8000":
            data['tolerance'] = 0.03
        else:
>           raise RuntimeError("Pls update this script for your platform.")
E           RuntimeError: Pls update this script for your platform.

**Comment by rraghav-cisco** [comment]
@echuawu : Yes, pls add your platform's tolerance value as required. You can add me as a reviewer if needed.

**Comment by echuawu** [comment]
@rraghav-cisco , I found a new PR(#8339), it has added tolerance for nvidia asic.

**Comment by yxieca** [comment]
@echuawu, @rraghav-cisco  Please raise PR to add the topology marker accordingly.

@wangxin can we add precheck to catch missing topology marker?

**Comment by wangxin** [comment]
Yes, adding precheck to catch such issue is a good idea. It is doable. I'll take care of it.

**Comment by echuawu** [comment]
@yxieca PR to add topology marker https://github.com/sonic-net/sonic-mgmt/pull/10705

**Comment by echuawu** [comment]
Hello @wangxin , there is a prepare testbed failure in the check of https://github.com/sonic-net/sonic-mgmt/pull/10705. Could you help check the reason?
